src/common/types/electron_store.ts

```diff
@@ -40,7 +41,7 @@ export interface StoreStructure {
       gogdlLogFile: string
       nileLogFile: string
     }
-    'window-props': Electron.Rectangle
+    'window-props': WindowProps
```

Comment:
In general, altering types of a store isn't that great of an idea (since the underlying data of course doesn't change, so you're still getting the old type even though you think you're getting the new one)
This isn't *that bad* here (the only missing prop. is "maximized", and it being `undefined` has the same effect as it just being `false`), but it's something to keep in mind

Reply:
Oh yes that's clear, normally we'd add some code that sort of migrates the config. But like you mentioned it's not required in this case